apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,40 @@
+import { AsyncPipe } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import { randomCity } from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { DataCardComponentBase, DataStoreBase } from '../../shared';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="data$ | async"
+      (addNewItem)="onAddNewItem()"
+      (deleteItem)="onDeleteItem($event)"
+      [backgroundColor]="'rgba(250, 10, 250, 0.1)'">
```

Comment:
I think this is better than using `::ng-deep`.

We could also think of allowing to pass a class-name: `backgroundClass="bg-red-200"`

This allows to reuse styles from an existing CSS-Framework like prime flex or tailwind.

`#idea` `#no-blocker`

Reply:
I've left it with `style`, but with refactored implementation class could be easily implemented as well :)